website/content/en/docs/advanced-topics/heterogeneous.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Heterogeneous Cluster Support
+linkTitle: Heterogeneous Cluster Support
+weight: 80
+---
+
+An operator deployed to a heterogeneous cluster may not support all of the node architectures in the cluster. This has scheduling implications for both the operator as well as its operands.
+
+### Setting Operator Default Architectures
+
+To ensure that the deployment only targets nodes of a supported architecture, the `ClusterServiceVersion` deployment spec should be updated as follows. For each of the deployments under `spec.install.deployments`, add this block to the `spec.template.spec`. 
+
+```yaml
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: kubernetes.io/arch
+          operator: In
+          values:
+          - amd64
+          - arm64
+          - ppc64le
+          - s390x
+```
+
+The list of architecture values should only include the architectures the operator supports. The full list of possible `GOARCH` values is [available here](https://github.com/golang/go/blob/master/src/go/build/syslist.go).
+
+### Setting Operand Default Architectures
+
+The affinity block shown above can also be used to set the default architectures for operands. Because operands can be created and configured many different ways, this section highlights the most common places where resources will need updates. As a general rule, anywhere in the operator where a `PodSpec` or `PodTemplateSpec` is defined, you'll want to supply a corresponding affinity block covering the supported arches. This includes pod definitions as well as objects that create collections of pods - such as deployments, daemonsets, statefulsets, etc.
+
+#### Go Operators
+``
+The most common directory to look at for objects that will need an architecture affinity block will is the operators ``config` directory. In the go samples app, located in the `testdata directory`, you'll find several Deployment configsthat need this updated including:
+`testdata/go/v2/memcached-operator/config/default/manager_auth_proxy_patch.yaml`
+`testdata/go/v2/memcached-operator/config/default/manager_webhook_patch.yaml`
+`testdata/go/v2/memcached-operator/config/manager/manager.yaml`
```

Comment:
We should not use go/v2 that is deprecated and should no longer be used.

Reply:
Updated to v3